docs/sources/setup-grafana/configure-security/configure-authentication/gitlab.md

```diff
@@ -132,6 +132,18 @@ role_attribute_path = is_admin && 'Admin' || 'Viewer'
 
 This allows every GitLab Admin to be an Admin in Grafana.
 
+#### Map roles using groups
+
+Groups can also be used to map roles. For instance,
+if you have a group called 'example-group' you can use the following snippet to
```

Comment:
nit: Could we use GitHub specific more real example, as I spent fare amount of time trying to understand why my setup is not working, until I have realized I am missing `@` at the beginning. So perhaps `@example-group` ? Or maybe even describe with human words how are we building the shorthand version [here](https://github.com/grafana/grafana/pull/52407/files#diff-5bd5381c7a8841520acb8a969432c1454f7b5888e00d8f68ef1a62807cfde436R244).